File: Ambulance.BLL/Models/HospitalModel.cs
using NetTopologySuite.Geometries;
using System;
using System.Collections.Generic;
namespace AmbulanceSystem.BLL.Models;

public class HospitalModel
{
    public int HospitalId { get; set; }
    public string? Name { get; set; }
    public Point? Location { get; set; }
    public string? Note { get; set; }
    public List<BrigadeModel>? Brigades { get; set; } = new List<BrigadeModel>();
    public List<CallModel>? Calls { get; set; }
}

File: public/Models/HospitalModel.js
import { BrigadeModel } from "./BrigadeModel.js";
import { CallModel } from "./CallModel.js";

export class HospitalModel {
    constructor({
        hospitalId = 0,
        name = "",
        latitude = null,
        longitude = null,
        note = "",
        brigades = [],
        calls = []
    } = {}) {
        this.hospitalId = hospitalId;
        this.name = name;
        this.latitude = latitude;
        this.longitude = longitude;
        this.note = note;
        this.brigades = brigades.map(b => new BrigadeModel(b));
        this.calls = calls.map(c => new CallModel(c));
    }
}